src/Language/PureScript/CoreFn/Desugar.hs

```diff
@@ -56,9 +56,6 @@ moduleToCoreFn env (A.Module modSS coms mn decls (Just exps)) =
   dedupeImports :: [(Ann, ModuleName)] -> [(Ann, ModuleName)]
   dedupeImports = fmap swap . M.toList . M.fromListWith const . fmap swap
 
-  ssA :: SourceSpan -> Ann
-  ssA ss = (ss, [], Nothing, Nothing)
-
```

Comment:
Note to self: this is removed because `Language.PureScript.CoreFn.Ann`, line 19, `ssAnn` is the same exact definition.